object_model/artifact.py

```diff
@@ -365,57 +365,179 @@ def pull(self, manifest={}):
 
         self.__commit__()
 
-    def peek(self, head=25, taip='default', func = lambda x: x):
-        trueVersioningDir = self.xp_state.versioningDirectory
-        self.xp_state.versioningDirectory = os.path.expanduser('~') + '/' + '1fdf8583bfd663e98918dea393e273cc'
 
-        # Create temporary versioning directory to store files
-        if os.path.exists(self.xp_state.versioningDirectory):
-            rmtree(self.xp_state.versioningDirectory)
-            os.mkdir(self.xp_state.versioningDirectory)
+    def peek(self, head=25, manifest={}, bindings = {}, func = lambda x: x):
+
+        self.xp_state.versioningDirectory = os.path.expanduser('~') + '/' + 'flor.d'
+
+        # Recreate the tmpexperiment directory
+        tmpexperiment = self.xp_state.tmpexperiment
```

Comment:
Let's switch to tempfile package for temporary directories. See `run` in https://github.com/ucbrise/flor/blob/master/headers.py